service/src/main/java/uk/gov/hmcts/reform/fpl/model/Others.java

```diff
@@ -1,16 +1,34 @@
 package uk.gov.hmcts.reform.fpl.model;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.google.common.collect.ImmutableList;
 import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Data;
 import uk.gov.hmcts.reform.fpl.model.common.Element;
 
 import java.util.List;
+import java.util.stream.Collectors;
 
 @Data
 @Builder
 @AllArgsConstructor
 public class Others {
     private final Other firstOther;
     private final List<Element<Other>> additionalOthers;
+
+    @JsonIgnore
+    public List<Other> getAllOthers() {
+        ImmutableList.Builder<Other> builder = ImmutableList.builder();
+
+        if (firstOther != null) {
```

Comment:
I think we can move it in single line `if (firstOther != null) { builder.add(firstOther); }`

Reply:
Linter does not like this :(